blockchain/beacon/src/beacon_entries.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use encoding::{
+    de::{self, Deserializer},
+    ser::{self, Serializer},
```

Comment:
```suggestion
    de::{Serialize, Deserializer},
    ser::{Deserialize, Serializer},
```
Why prefix if only using those?